<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.42.1" />
  <meta name="author" content="Nathan Skene">

  
  
  
  
  <meta name="description" content="Postdoctoral Researcher in Hjerling-Leffler Molecular Neurobiology Lab">

  
  <link rel="alternate" hreflang="en-us" href="https://NathanSkene.github.io/public/">

  


  

  
  
  <meta name="theme-color" content="#0095eb">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat:400,700%7cRoboto:400,400italic,700%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/public/styles.css">
  

  

  
  <link rel="alternate" href="https://NathanSkene.github.io/public/index.xml" type="application/rss+xml" title="NATHAN SKENE">
  <link rel="feed" href="https://NathanSkene.github.io/public/index.xml" type="application/rss+xml" title="NATHAN SKENE">
  

  <link rel="manifest" href="/public/site.webmanifest">
  <link rel="icon" type="image/png" href="/public/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/public/img/icon-192.png">

  <link rel="canonical" href="https://NathanSkene.github.io/public/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@n_skene">
  <meta property="twitter:creator" content="@n_skene">
  
  <meta property="og:site_name" content="NATHAN SKENE">
  <meta property="og:url" content="https://NathanSkene.github.io/public/">
  <meta property="og:title" content="NATHAN SKENE">
  <meta property="og:description" content="Postdoctoral Researcher in Hjerling-Leffler Molecular Neurobiology Lab">
  <meta property="og:locale" content="en-us">
  
  <meta property="og:updated_time" content="2016-04-20T00:00:00&#43;02:00">
  

  
  

  <title>NATHAN SKENE</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/public/">NATHAN SKENE</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/public/#about" data-target="#about">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/public/#publications_selected" data-target="#publications_selected">
            
            <span>Publications</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/public/#software" data-target="#software">
            
            <span>Software</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/public/#contact" data-target="#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>



<span id="homepage" style="display: none"></span>


  





  
  
  
  <section id="about" class="home-section">
    <div class="container">
      



<div class="row" itemprop="author" itemscope itemtype="http://schema.org/Person" itemref="person-email">
  <div class="col-xs-12 col-md-4">
    <div id="profile">

      
      <div class="portrait" style="background-image: url('/public/img/Jens_Grupp_ZIM_6681_square.jpg');"></div>
      <meta itemprop="image" content="https://NathanSkene.github.io/public/img/Jens_Grupp_ZIM_6681_square.jpg">
      

      <div class="portrait-title">
        <h2 itemprop="name">Nathan Skene</h2>
        <h3 itemprop="jobTitle">Postdoctoral Researcher in Hjerling-Leffler Molecular Neurobiology Lab</h3>

        
        

        
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          <a href="http://www.hjerling-leffler-lab.org/" target="_blank" itemprop="url" rel="noopener">
          <span itemprop="name">Hjerling-Leffler lab</span>
          </a>
        </h3>
        
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          <a href="http://ki.se/en/startpage" target="_blank" itemprop="url" rel="noopener">
          <span itemprop="name">Karolinska Institute</span>
          </a>
        </h3>
        
      </div>

      <link itemprop="url" href="https://NathanSkene.github.io/public/">

      <ul class="network-icon" aria-hidden="true">
        
        
        <li>
          <a itemprop="sameAs" href="mailto:nathan.skene@ki.se" target="_blank" rel="noopener">
            <i class="fa fa-envelope big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="//twitter.com/n_skene" target="_blank" rel="noopener">
            <i class="fa fa-twitter big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="https://scholar.google.co.uk/citations?user=nu6vg78AAAAJ&amp;hl=en&amp;oi=ao" target="_blank" rel="noopener">
            <i class="ai ai-google-scholar big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="//github.com/NathanSkene" target="_blank" rel="noopener">
            <i class="fa fa-github big-icon"></i>
          </a>
        </li>
        
      </ul>

    </div>
  </div>
  <div class="col-xs-12 col-md-8" itemprop="description">

    

<h1 id="biography">Biography</h1>

<p>I am a postdoctoral researcher at the Karolinska Institutet where I works with <a href="http://www.hjerling-leffler-lab.org/">Jens Hjerling-Leffler</a>. My interests lie in using human genetics to gain insight into the neurobiology of psychiatric disorders and cognitive traits. Currently I am focused on identifying the causal cell types underlying complex genetic traits.</p>

<p>I gained an undergraduate degree in Artificial Intelligence and Cybernetics from the University of Reading in 2008, followed by an MPhil at the University of Cambridge in Computational Biology in 2009.  I went onto do a PhD in Molecular Biology at the Wellcome Trust Sanger Institute working with <a href="https://www.ed.ac.uk/clinical-brain-sciences/people/principal-investigators/professor-seth-grant">Prof Seth Grant</a>. During this time I worked on the <a href="http://www.genes2cognition.org/">Genes to Cognition programme</a>, analysing the transcriptomic changes seen in a mice carrying a wide range of synaptic mutations.</p>

<p>Following my PhD I worked at the University of Edinburgh, where I studied how postnatal gene expression changes influence the <a href="https://elifesciences.org/articles/17915">onset</a> of psychiatric disorders. At this time I developed a method (<a href="https://github.com/NathanSkene/EWCE">Expression Weighted Celltype Enrichment</a>) which made it possible to show that <a href="https://www.frontiersin.org/articles/10.3389/fnins.2016.00016">microglia</a> are the celltype whose expression profile best explains the rare &amp; common variant&rsquo;s associated with Alzheimer&rsquo;s disease.</p>

<p>I then begun a Wellcome Trust funded postdoctoral position as part of the <a href="http://www.ucl.ac.uk/cortexlab/neuromics">Functional Neuromics</a> project split between University College London and the Karolinska Institutet. At KI we adapted single cell disassociation protocols such that we could obtain sufficient numbers of healthy interneurons from adult mice that we could catalogue the full complement of GABAergic cell types found in the <a href="journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2006387">CA1</a> region of the hippocampus. To enable greater insight into which of these cell types play a role in human disease/cognition we begun collaborating with <a href="https://ki.se/en/people/patrsu">Patrick Sullivan&rsquo;s</a> psychiatric genetics team: together we established the methods which enabled us to map the cell types underlying <a href="https://www.nature.com/articles/s41588-018-0129-5">Schizophrenia, Major Depression</a>, <a href="https://www.nature.com/articles/s41588-018-0152-6">IQ</a>, <a href="https://www.nature.com/articles/s41588-018-0151-7">Neuroticism</a>, <a href="https://www.biorxiv.org/content/early/2018/02/22/258533">Alzheimer&rsquo;s</a> and <a href="https://www.biorxiv.org/content/early/2018/02/01/214973">Insomnia</a>. We released the <a href="https://github.com/NathanSkene/MAGMA_Celltyping">MAGMA_Celltyping</a> package to enable others to apply this method to future GWAS studies. The KI single cell superset data which we used for cell type mapping is available <a href="www.hjerling-leffler-lab.org/data/scz_singlecell/">here</a>.</p>


    <div class="row">

      
      <div class="col-sm-5">
        <h3>Interests</h3>
        <ul class="ul-interests">
          
          <li>Neurobiology</li>
          
          <li>Human Genetics</li>
          
          <li>Psychiatric Disorders</li>
          
          <li>Transcriptomics</li>
          
          <li>Bioinformatics</li>
          
        </ul>
      </div>
      

      
      <div class="col-sm-7">
        <h3>Education</h3>
        <ul class="ul-edu fa-ul">
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">PhD in Molecular Biology, 2014</p>
              <p class="institution">Wellcome Trust Sanger Institute, University of Cambridge</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">MPhil in Computational Biology, 2009</p>
              <p class="institution">University of Cambridge</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">BSc in Artificial Intelligence and Cybernetics, 2008</p>
              <p class="institution">University of Reading</p>
            </div>
          </li>
          
        </ul>
      </div>
      

    </div>
  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="publications_selected" class="home-section">
    <div class="container">
      



<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Selected Publications</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
      
    

    
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/spatialatlas/">
      
      
        <img src="/public/img/headers/spatialAtlas.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/spatialatlas/" itemprop="url">A spatial atlas of inhibitory cell types in mouse hippocampus</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Understanding the function of a tissue requires knowing the spatial organization of its constituent cell types. In the cerebral cortex, single-cell RNA sequencing (scRNA-seq) has revealed the genome-wide expression patterns that define its many, closely related cell types, but cannot reveal their spatial arrangement. Here we introduce probabilistic cell typing by in situ sequencing (pciSeq), an approach that leverages prior scRNA-seq classification to identify cell types using multiplexed in situ RNA detection. We applied this method to map the inhibitory neurons of hippocampal area CA1, a cell system critical for memory function, for which ground truth is available from extensive prior work identifying the laminar organization of subtly differing cell types. Our method confidently identified 16 interneuron classes, in a spatial arrangement closely matching ground truth. This method will allow identifying the spatial organization of fine cell types across the brain and other tissues.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Xiaoyan Qian, Kenneth D Harris, Thomas Hauling, Dimitris Nicoloutsopoulos, Ana Munoz Manchado, Nathan Skene, Jens Hjerling-Leffler, Mats Nilsson
    
  </div>

  <div class="pub-publication">
    
    bioRxiv,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/10/01/431957" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/striatumscrna/">
      
      
        <img src="/public/img/headers/striatumSCRNA.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/striatumscrna/" itemprop="url">Diversity of Interneurons in the Dorsal Striatum Revealed by Single-Cell RNA Sequencing and PatchSeq</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Striatal locally projecting neurons, or interneurons, act on nearby circuits and shape functional output to the rest of the basal ganglia. We performed single-cell RNA sequencing of striatal cells enriching for interneurons. We find seven discrete interneuron types, six of which are GABAergic. In addition to providing specific markers for the populations previously described, including those expressing Sst/Npy, Th, Npy without Sst, and Chat, we identify two small populations of cells expressing Cck with or without Vip. Surprisingly, the Pvalb-expressing cells do not constitute a discrete cluster but rather are part of a larger group of cells expressing Pthlh with a spatial gradient of Pvalb expression. Using PatchSeq, we show that Pthlh cells exhibit a continuum of electrophysiological properties correlated with expression of Pvalb. Furthermore, we find significant molecular differences that correlate with differences in electrophysiological properties between Pvalb-expressing cells of the striatum and those of the cortex.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    A Muñoz-Manchado, C Gonzales, A Zeisel, H Munguba, B Bekkouche, N Skene, P Lönnerberg, J Ryge, K Harris, S Linnarsson, J Hjerling-Leffler
    
  </div>

  <div class="pub-publication">
    
    Cell Reports,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.cell.com/cell-reports/fulltext/S2211-1247%2818%2931156-2" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/moleculararch/">
      
      
        <img src="/public/img/headers/molArch.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/moleculararch/" itemprop="url">Molecular architecture of the mouse nervous system</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    The mammalian nervous system executes complex behaviors controlled by specialized, precisely positioned, and interacting cell types. Here, we used RNA sequencing of half a million single cells to create a detailed census of cell types in the mouse nervous system. We mapped cell types spatially and derived a hierarchical, data-driven taxonomy. Neurons were the most diverse and were grouped by developmental anatomical units and by the expression of neurotransmitters and neuropeptides. Neuronal diversity was driven by genes encoding cell identity, synaptic connectivity, neurotransmission, and membrane conductance. We discovered seven distinct, regionally restricted astrocyte types that obeyed developmental boundaries and correlated with the spatial distribution of key glutamate and glycine neurotransmitters. In contrast, oligodendrocytes showed a loss of regional identity followed by a secondary diversification. The resource presented here lays a solid foundation for understanding the molecular architecture of the mammalian nervous system and enables genetic manipulation of specific cell types.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Amit Zeisel, Hannah Hochgerner, Peter Lonnerberg, Anna Johnsson, Fatima Memic, Job van der Zwan, Martin Haring, Emelie Braun, Lars Borm, Gioele La Manno, Simone Codeluppi, Alessandro Furlan, Nathan Skene, Kenneth D Harris, Jens Hjerling Leffler, Ernest Arenas, Patrik Ernfors, Ulrika Marklund, Sten Linnarsson
    
  </div>

  <div class="pub-publication">
    
    Cell,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.sciencedirect.com/science/article/pii/S009286741830789X?via%3Dihub" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/architecture-of-the-mouse-brain-synaptome/">
      
      
        <img src="/public/img/headers/synaptomeArchitecture.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/architecture-of-the-mouse-brain-synaptome/" itemprop="url">Architecture of the Mouse Brain Synaptome</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Synapses are found in vast numbers in the brain and contain complex proteomes. We developed genetic labeling and imaging methods to examine synaptic proteins in individual excitatory synapses across all regions of the mouse brain. Synapse catalogs were generated from the molecular and morphological features of a billion synapses. Each synapse subtype showed a unique anatomical distribution, and each brain region showed a distinct signature of synapse subtypes. Whole-brain synaptome cartography revealed spatial architecture from dendritic to global systems levels and previously unknown anatomical features. Synaptome mapping of circuits showed correspondence between synapse diversity and structural and functional connectomes. Behaviorally relevant patterns of neuronal activity trigger spatiotemporal postsynaptic responses sensitive to the structure of synaptome maps. Areas controlling higher cognitive function contain the greatest synapse diversity, and mutations causing cognitive disorders reorganized synaptome maps. Synaptome technology and resources have wide-ranging application in studies of the normal and diseased brain.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    F Zhu, M Cizeron, Z Qiu, R Benavides-Piccione, M Kopanitsa, N Skene, J DeFelipe, E Fransén, N Komiyama, S Grant
    
  </div>

  <div class="pub-publication">
    
    Neuron,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.cell.com/neuron/fulltext/S0896-6273%2818%2930581-6" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/savage-iq-2018/">
      
      
        <img src="/public/img/headers/iqSavage.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/savage-iq-2018/" itemprop="url">Genome-wide association meta-analysis in 269,867 individuals identifies new genetic and functional links to intelligence</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Intelligence is highly heritable and a major determinant of human health and well-being. Recent genome-wide meta-analyses have identified 24 genomic loci linked to variation in intelligence, but much about its genetic underpinnings remains to be discovered. Here, we present a large-scale genetic association study of intelligence (n=269,867), identifying 205 associated genomic loci (190 new) and 1,016 genes (939 new) via positional mapping, expression quantitative trait locus (eQTL) mapping, chromatin interaction mapping, and gene-based association analysis. We find enrichment of genetic effects in conserved and coding regions and associations with 146 nonsynonymous exonic variants. Associated genes are strongly expressed in the brain, specifically in striatal medium spiny neurons and hippocampal pyramidal neurons. Gene set analyses implicate pathways related to nervous system development and synaptic structure. We confirm previous strong genetic correlations with multiple health-related outcomes, and Mendelian randomization analysis results suggest protective effects of intelligence for Alzheimer’s disease and ADHD and bidirectional causation with pleiotropic effects for schizophrenia. These results are a major step forward in understanding the neurobiology of cognitive function as well as genetically related neurological and psychiatric disorders.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Jeanne E. Savage, Philip R. Jansen, Sven Stringer, Kyoko Watanabe, Julien Bryois, Christiaan A. de Leeuw, Mats Nagel, Swapnil Awasthi, Peter B. Barr, Jonathan R. I. Coleman, Katrina L. Grasby, Anke R. Hammerschlag, Jakob A. Kaminski, Robert Karlsson, Eva Krapohl, Max Lam, Marianne Nygaard, Chandra A. Reynolds, Joey W. Trampush, Hannah Young, Delilah Zabaneh, Sara Hägg, Narelle K. Hansell, Ida K. Karlsson, Sten Linnarsson, Grant W. Montgomery, Ana B. Muñoz-Manchado, Erin B. Quinlan, Gunter Schumann, Nathan G. Skene, Bradley T. Webb, Tonya White, Dan E. Arking, Dimitrios Avramopoulos, Robert M. Bilder, Panos Bitsios, Katherine E. Burdick, Tyrone D. Cannon, Ornit Chiba-Falek, Andrea Christoforou, Elizabeth T. Cirulli, Eliza Congdon, Aiden Corvin, Gail Davies, Ian J. Deary, Pamela DeRosse, Dwight Dickinson, Srdjan Djurovic, Gary Donohoe, Emily Drabant Conley, Johan G. Eriksson, Thomas Espeseth, Nelson A. Freimer, Stella Giakoumaki, Ina Giegling, Michael Gill, David C. Glahn, Ahmad R. Hariri, Alex Hatzimanolis, Matthew C. Keller, Emma Knowles, Deborah Koltai, Bettina Konte, Jari Lahti, Stephanie Le Hellard, Todd Lencz, David C. Liewald, Edythe London, Astri J. Lundervold, Anil K. Malhotra, Ingrid Melle, Derek Morris, Anna C. Need, William Ollier, Aarno Palotie, Antony Payton, Neil Pendleton, Russell A. Poldrack, Katri Räikkönen, Ivar Reinvang, Panos Roussos, Dan Rujescu, Fred W. Sabb, Matthew A. Scult, Olav B. Smeland, Nikolaos Smyrnis, John M. Starr, Vidar M. Steen, Nikos C. Stefanis, Richard E. Straub, Kjetil Sundet, Henning Tiemeier, Aristotle N. Voineskos, Daniel R. Weinberger, Elisabeth Widen, Jin Yu, Goncalo Abecasis, Ole A. Andreassen, Gerome Breen, Lene Christiansen, Birgit Debrabant, Danielle M. Dick, Andreas Heinz, Jens Hjerling-Leffler, M. Arfan Ikram, Kenneth S. Kendler, Nicholas G. Martin, Sarah E. Medland, Nancy L. Pedersen, Robert Plomin, Tinca J. C. Polderman, Stephan Ripke, Sophie van der Sluis, Patrick F. Sullivan, Scott I. Vrieze, Margaret J. Wright &amp; Danielle Posthuma
    
  </div>

  <div class="pub-publication">
    
    Nature Genetics,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41588-018-0151-7" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/nagal-neuroticism-2018/">
      
      
        <img src="/public/img/headers/neuroticism.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/nagal-neuroticism-2018/" itemprop="url">Meta-analysis of genome-wide association studies for neuroticism in 449,484 individuals identifies novel genetic loci and pathways</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Neuroticism is an important risk factor for psychiatric traits, including depression1, anxiety2,3, and schizophrenia4,5,6. At the time of analysis, previous genome-wide association studies (GWAS) reported 16 genomic loci associated to neuroticism. Here we conducted a large GWAS meta-analysis (n=449,484) of neuroticism and identified 136 independent genome-wide significant loci (124 new at the time of analysis), which implicate 599 genes. Functional follow-up analyses showed enrichment in several brain regions and involvement of specific cell types, including dopaminergic neuroblasts (P=3.49×10−8), medium spiny neurons (P=4.23×10−8), and serotonergic neurons (P=1.37×10−7). Gene set analyses implicated three specific pathways: neurogenesis (P=4.43×10−9), behavioral response to cocaine processes (P=1.84×10−7), and axon part (P=5.26×10−8). We show that neuroticism’s genetic signal partly originates in two genetically distinguishable subclusters (‘depressed affect’ and ‘worry’), suggesting distinct causal mechanisms for subtypes of individuals. Mendelian randomization analysis showed unidirectional and bidirectional effects between neuroticism and multiple psychiatric traits. These results enhance neurobiological understanding of neuroticism and provide specific leads for functional follow-up experiments.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Mats Nagel, Philip R. Jansen, Sven Stringer, Kyoko Watanabe, Christiaan A. de Leeuw, Julien Bryois, Jeanne E. Savage, Anke R. Hammerschlag, Nathan G. Skene, Ana B. Muñoz-Manchado, 23andMe Research Team, Tonya White, Henning Tiemeier, Sten Linnarsson, Jens Hjerling-Leffler, Tinca J. C. Polderman, Patrick F. Sullivan, Sophie van der Sluis &amp; Danielle Posthuma
    
  </div>

  <div class="pub-publication">
    
    Nature Genetics,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41588-018-0151-7" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/kenharris-2018-ca1/">
      
      
        <img src="/public/img/headers/KenHarrisCA1.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/kenharris-2018-ca1/" itemprop="url">Classes and continua of hippocampal CA1 inhibitory neurons revealed by single-cell transcriptomics</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Understanding any brain circuit will require a categorization of its constituent neurons. In hippocampal area CA1, at least 23 classes of GABAergic neuron have been proposed to date. However, this list may be incomplete; additionally, it is unclear whether discrete classes are sufficient to describe the diversity of cortical inhibitory neurons or whether continuous modes of variability are also required. We studied the transcriptomes of 3,663 CA1 inhibitory cells, revealing 10 major GABAergic groups that divided into 49 fine-scale clusters. All previously described and several novel cell classes were identified, with three previously described classes unexpectedly found to be identical. A division into discrete classes, however, was not sufficient to describe the diversity of these cells, as continuous variation also occurred between and within classes. Latent factor analysis revealed that a single continuous variable could predict the expression levels of several genes, which correlated similarly with it across multiple cell types. Analysis of the genes correlating with this variable suggested it reflects a range from metabolically highly active faster-spiking cells that proximally target pyramidal cells to slower-spiking cells targeting distal dendrites or interneurons. These results elucidate the complexity of inhibitory neurons in one of the simplest cortical structures and show that characterizing these cells requires continuous modes of variation as well as discrete cell classes.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Kenneth D Harris, Hannah Hochgerner, Nathan G Skene, Lorenza Magno, Linda Katona, Carolina Bengtsson Gonzales, Peter Somogyi, Nicoletta Kessaris, Sten Linnarsson, Jens Hjerling-Leffler
    
  </div>

  <div class="pub-publication">
    
    PLoS biology,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41588-018-0151-7" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/skene-natgen-2018/">
      
      
        <img src="/public/img/headers/schizNatGen.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/skene-natgen-2018/" itemprop="url">Genetic identification of brain cell types underlying schizophrenia</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    With few exceptions, the marked advances in knowledge about the genetic basis of schizophrenia have not converged on findings that can be confidently used for precise experimental modeling. By applying knowledge of the cellular taxonomy of the brain from single-cell RNA sequencing, we evaluated whether the genomic loci implicated in schizophrenia map onto specific brain cell types. We found that the common-variant genomic results consistently mapped to pyramidal cells, medium spiny neurons (MSNs) and certain interneurons, but far less consistently to embryonic, progenitor or glial cells. These enrichments were due to sets of genes that were specifically expressed in each of these cell types. We also found that many of the diverse gene sets previously associated with schizophrenia (genes involved in synaptic function, those encoding mRNAs that interact with FMRP, antipsychotic targets, etc.) generally implicated the same brain cell types. Our results suggest a parsimonious explanation: the common-variant genetic results for schizophrenia point at a limited set of neurons, and the gene sets point to the same cells. The genetic risk associated with MSNs did not overlap with that of glutamatergic pyramidal cells and interneurons, suggesting that different cell types have biologically distinct roles in schizophrenia.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Nathan G. Skene, Julien Bryois, Trygve E. Bakken, Gerome Breen, James J Crowley, Héléna A Gaspar, Paola Giusti-Rodriguez, Rebecca D Hodge, Jeremy A. Miller, Ana Muñoz-Manchado, Michael C O’Donovan, Michael J Owen, Antonio F Pardiñas, Jesper Ryge, James T R Walters, Sten Linnarsson, Ed S. Lein, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, Patrick F Sullivan, Jens Hjerling-Leffler
    
  </div>

  <div class="pub-publication">
    
    Nature Genetics,
    
    2018
  </div>

  <div class="pub-links">
    



<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2017/06/02/145466" target="_blank" rel="noopener">
  Preprint
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41588-018-0129-5.pdf" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/geromeiq-molpsy-2018/">
      
      
        <img src="/public/img/headers/bioPsychIQ.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/geromeiq-molpsy-2018/" itemprop="url">Biological annotation of genetic loci associated with intelligence in a meta-analysis of 87,740 individuals</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Variance in IQ is associated with a wide range of health outcomes, and 1% of the population are affected by intellectual disability. Despite a century of research, the fundamental neural underpinnings of intelligence remain unclear. We integrate results from genome-wide association studies (GWAS) of intelligence with brain tissue and single cell gene expression data to identify tissues and cell types associated with intelligence. GWAS data for IQ (N = 78,308) were meta-analyzed with a study comparing 1247 individuals with mean IQ ~170 to 8185 controls. Genes associated with intelligence implicate pyramidal neurons of the somatosensory cortex and CA1 region of the hippocampus, and midbrain embryonic GABAergic neurons. Tissue-specific analyses find the most significant enrichment for frontal cortex brain expressed genes. These results suggest specific neuronal cell types and genes may be involved in intelligence and provide new hypotheses for neuroscience experiments using model systems.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Jonathan R. I. Coleman, Julien Bryois, Héléna A. Gaspar, Philip R. Jansen, Jeanne E. Savage, Nathan Skene, Robert Plomin, Ana B. Muñoz-Manchado, Sten Linnarsson, Greg Crawford, Jens Hjerling-Leffler, Patrick F. Sullivan, Danielle Posthuma, Gerome Breen
    
  </div>

  <div class="pub-publication">
    
    Molecular Psychiatry,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41380-018-0040-6" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/gwasalzh/">
      
      
        <img src="/public/img/headers/gwasAlzh.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/gwasalzh/" itemprop="url">Genetic meta-analysis identifies 9 novel loci and functional pathways for Alzheimers disease risk</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Late onset Alzheimer&rsquo;s disease (AD) is the most common form of dementia with more than 35 million people affected worldwide, and no curative treatment available. AD is highly heritable and recent genome-wide meta-analyses have identified over 20 genomic loci associated with AD, yet only explaining a small proportion of the genetic variance indicating that undiscovered loci exist. Here, we performed the largest genome-wide association study of clinically diagnosed AD and AD-by-proxy (71,880 AD cases, 383,378 controls). AD-by-proxy status is based on parental AD diagnosis, and showed strong genetic correlation with AD (rg=0.81). Genetic meta-analysis identified 29 risk loci, of which 9 are novel, and implicating 215 potential causative genes. Independent replication further supports these novel loci in AD. Associated genes are strongly expressed in immune-related tissues and cell types (spleen, liver and microglia). Furthermore, gene-set analyses indicate the genetic contribution of biological mechanisms involved in lipid-related processes and degradation of amyloid precursor proteins. We show strong genetic correlations with multiple health-related outcomes, and Mendelian randomisation results suggest a protective effect of cognitive ability on AD risk. These results are a step forward in identifying more of the genetic factors that contribute to AD risk and add novel insights into the neurobiology of AD to guide new drug development.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    I Jansen, J Savage, K Watanabe, J Bryois, D Williams, S Steinberg, J Sealock, I Karlsson, S Hagg, L Athanasiu, N Voyle, P Proitsi, A Witoelar, S Stringer, D Aarsland, I Almdahl, F Andersen, S Bergh, F Bettella, S Bjornsson, A Braekhus, G Brathen, C Leeuw, R Desikan, S Djurovic, L Dumitrescu, T Fladby, T Homan, P Jonsson, A Rongve, I Saltvedt, S Sando, G Selbak, N Skene, J Snaedal, E Stordal, I Ulstein, Y Wang, L White, J Hjerling-Leffler, P Sullivan, W van der Flier, R Dobson, L Davis, H Stefansson, K Stefansson, N Pedersen, S Ripke, O Andreassen, D Posthuma
    
  </div>

  <div class="pub-publication">
    
    bioRxiv,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/02/22/258533" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/gwasinsomnia/">
      
      
        <img src="/public/img/headers/gwasInsomnia.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/gwasinsomnia/" itemprop="url">Genome-Wide Meta-Analysis of Insomnia Complaints (N=1,331,010) Identifies Novel Genetic Loci and Functional Pathways</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Insomnia is the second-most prevalent mental disorder, with no sufficient treatment available. Despite a substantial role of genetic factors, only a handful of genes have been implicated and insight into the associated neurobiological pathways remains limited. Here, we use an unprecedented large genetic association sample (N=1,331,010) to allow detection of a substantial number of genetic variants and gain insight into biological functions, cell types and tissues involved in insomnia. We identify 202 genome-wide significant loci implicating 956 genes through positional, eQTL and chromatin interaction mapping. We show involvement of the axonal part of neurons, of specific cortical and subcortical tissues, and of two specific cell-types in insomnia: striatal medium spiny neurons and hypothalamic neurons. These cell-types have been implicated previously in the regulation of reward processing, sleep and arousal in animal studies, but have never been genetically linked to insomnia in humans. We found weak genetic correlations with other sleep-related traits, but strong genetic correlations with psychiatric and metabolic traits. Mendelian randomization identified causal effects of insomnia on specific psychiatric and metabolic traits. Our findings reveal key brain areas and cells implicated in the neurobiology of insomnia and its related disorders, and provide novel targets for treatment.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Philip R Jansen, Kyoko Watanabe, Sven Stringer, Nathan Skene, Julien Bryois, Anke R Hammerschlag, Chrstiaan A de Leeuw, Jeroen Benjamins, Ana B Munoz-Manchado, Mats Nagel, Jeanne E Savage, Henning Tiemeier, Tonya White, Joyce Y Tung, David A Hinds, Vladimir Vacic, Patrick F Sullivan, Sophie van der Sluis, Tinca JC Polderman, August B Smit, Jens Hjerling-Leffler, Eus JW van Someren, Danielle Posthuma
    
  </div>

  <div class="pub-publication">
    
    bioRxiv,
    
    2018
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/02/01/214973" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/marcia-proteome-2018/">
      
      
        <img src="/public/img/headers/smoking.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/marcia-proteome-2018/" itemprop="url">Proteomic analysis of postsynaptic proteins in regions of the human neocortex</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    The postsynaptic proteome of excitatory synapses comprises ~1,000 highly conserved proteins that control the behavioral repertoire, and mutations disrupting their function cause &gt;130 brain diseases. Here, we document the composition of postsynaptic proteomes in human neocortical regions and integrate it with genetic, functional and structural magnetic resonance imaging, positron emission tomography imaging, and behavioral data. Neocortical regions show signatures of expression of individual proteins, protein complexes, biochemical and metabolic pathways. We characterized the compositional signatures in brain regions involved with language, emotion and memory functions. Integrating large-scale GWAS with regional proteome data identifies the same cortical region for smoking behavior as found with fMRI data. The neocortical postsynaptic proteome data resource can be used to link genetics to brain imaging and behavior, and to study the role of postsynaptic proteins in localization of brain functions.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Marcia Roy, Oksana Sorokina, Nathan Skene, Clémence Simonnet, Francesca Mazzo, Ruud Zwart, Emanuele Sher, Colin Smith, J. Douglas Armstrong, Seth G. N. Grant
    
  </div>

  <div class="pub-publication">
    
    Nature Neuroscience,
    
    2017
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41593-017-0025-9" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/skene-elife-2017/">
      
      
        <img src="/public/img/headers/elifeALIGET.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/skene-elife-2017/" itemprop="url">A genomic lifespan program that reorganises the young adult brain is targeted in schizophrenia</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    The genetic mechanisms regulating the brain and behaviour across the lifespan are poorly understood. We found that lifespan transcriptome trajectories describe a calendar of gene regulatory events in the brain of humans and mice. Transcriptome trajectories defined a sequence of gene expression changes in neuronal, glial and endothelial cell-types, which enabled prediction of age from tissue samples. A major lifespan landmark was the peak change in trajectories occurring in humans at 26 years and in mice at 5 months of age. This species-conserved peak was delayed in females and marked a reorganization of expression of synaptic and schizophrenia-susceptibility genes. The lifespan calendar predicted the characteristic age of onset in young adults and sex differences in schizophrenia. We propose a genomic program generates a lifespan calendar of gene regulation that times age-dependent molecular organization of the brain and mutations that interrupt the program in young adults cause schizophrenia.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Nathan G Skene, Marcia Roy, Seth GN Grant
    
  </div>

  <div class="pub-publication">
    
    eLife,
    
    2017
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://elifesciences.org/articles/17915" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/ewce-2016/">
      
      
        <img src="/public/img/headers/ewce.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/ewce-2016/" itemprop="url">Identification of Vulnerable Cell Types in Major Brain Disorders Using Single Cell Transcriptomes and Expression Weighted Cell Type Enrichment</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    The cell types that trigger the primary pathology in many brain diseases remain largely unknown. One route to understanding the primary pathological cell type for a particular disease is to identify the cells expressing susceptibility genes. Although this is straightforward for monogenic conditions where the causative mutation may alter expression of a cell type specific marker, methods are required for the common polygenic disorders. We developed the Expression Weighted Cell Type Enrichment (EWCE) method that uses single cell transcriptomes to generate the probability distribution associated with a gene list having an average level of expression within a cell type. Following validation, we applied EWCE to human genetic data from cases of epilepsy, Schizophrenia, Autism, Intellectual Disability, Alzheimer&rsquo;s disease, Multiple Sclerosis and anxiety disorders. Genetic susceptibility primarily affected microglia in Alzheimer&rsquo;s and Multiple Sclerosis; was shared between interneurons and pyramidal neurons in Autism and Schizophrenia; while intellectual disabilities and epilepsy were attributable to a range of cell-types, with the strongest enrichment in interneurons. We hypothesized that the primary cell type pathology could trigger secondary changes in other cell types and these could be detected by applying EWCE to transcriptome data from diseased tissue. In Autism, Schizophrenia and Alzheimer&rsquo;s disease we find evidence of pathological changes in all of the major brain cell types. These findings give novel insight into the cellular origins and progression in common brain disorders. The methods can be applied to any tissue and disorder and have applications in validating mouse models.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Nathan G Skene, Seth GN Grant
    
  </div>

  <div class="pub-publication">
    
    Frontiers in Neuroscience,
    
    2016
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.frontiersin.org/articles/10.3389/fnins.2016.00016/full" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    
      
        <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
    
  
  
  <div>
    <a href="https://NathanSkene.github.io/public/publication/tnik-2012/">
      
      
        <img src="/public/img/headers/tnik.jpg" class="article-banner" itemprop="image">
      
    </a>
  </div>
  

  <h3 class="article-title" itemprop="name">
    <a href="https://NathanSkene.github.io/public/publication/tnik-2012/" itemprop="url">TNiK Is Required for Postsynaptic and Nuclear Signaling Pathways and Cognitive Function</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Traf2 and NcK interacting kinase (TNiK) contains serine-threonine kinase and scaffold domains and has been implicated in cell proliferation and glutamate receptor regulation in vitro. Here we report its role in vivo using mice carrying a knock-out mutation. TNiK binds protein complexes in the synapse linking it to the NMDA receptor (NMDAR) via AKAP9. NMDAR and metabotropic receptors bidirectionally regulate TNiK phosphorylation and TNiK is required for AMPA expression and synaptic function. TNiK also organizes nuclear complexes and in the absence of TNiK, there was a marked elevation in GSK3β and phosphorylation levels of its cognate phosphorylation sites on NeuroD1 with alterations in Wnt pathway signaling. We observed impairments in dentate gyrus neurogenesis in TNiK knock-out mice and cognitive testing using the touchscreen apparatus revealed impairments in pattern separation on a test of spatial discrimination. Object-location paired associate learning, which is dependent on glutamatergic signaling, was also impaired. Additionally, TNiK knock-out mice displayed hyperlocomotor behavior that could be rapidly reversed by GSK3β inhibitors, indicating the potential for pharmacological rescue of a behavioral phenotype. These data establish TNiK as a critical regulator of cognitive functions and suggest it may play a regulatory role in diseases impacting on its interacting proteins and complexes.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Marcelo P. Coba, Noboru H. Komiyama, Jess Nithianantharajah, Maksym V. Kopanitsa, Tim Indersmitten, Nathan G. Skene, Ellie J. Tuck, David G. Fricker, Kathryn A. Elsegood, Lianne E. Stanford, Nurudeen O. Afinowi, Lisa M. Saksida, Timothy J. Bussey, Thomas J. O&rsquo;Dell, Seth G.N. Grant
    
  </div>

  <div class="pub-publication">
    
    Journal of Neuroscience,
    
    2012
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="http://www.jneurosci.org/content/32/40/13987.short" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>


  </div>

</div>

      
    

  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="publications" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Recent Publications</h1>
    
    
    <p class="view-all">
      <a href="https://NathanSkene.github.io/public/publication/">
        More Publications
        <i class="fa fa-angle-double-right"></i>
      </a>
    </p>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
      
    

    

    
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    Xiaoyan Qian, Kenneth D Harris, Thomas Hauling, Dimitris Nicoloutsopoulos, Ana Munoz Manchado, Nathan Skene, Jens Hjerling-Leffler, Mats Nilsson</span>.
  <a href="https://NathanSkene.github.io/public/publication/spatialatlas/" itemprop="name">A spatial atlas of inhibitory cell types in mouse hippocampus</a>.
  bioRxiv,
  2018.
  <p>




<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/10/01/431957" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    A Muñoz-Manchado, C Gonzales, A Zeisel, H Munguba, B Bekkouche, N Skene, P Lönnerberg, J Ryge, K Harris, S Linnarsson, J Hjerling-Leffler</span>.
  <a href="https://NathanSkene.github.io/public/publication/striatumscrna/" itemprop="name">Diversity of Interneurons in the Dorsal Striatum Revealed by Single-Cell RNA Sequencing and PatchSeq</a>.
  Cell Reports,
  2018.
  <p>




<a class="btn btn-primary btn-outline btn-xs" href="https://www.cell.com/cell-reports/fulltext/S2211-1247%2818%2931156-2" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    Amit Zeisel, Hannah Hochgerner, Peter Lonnerberg, Anna Johnsson, Fatima Memic, Job van der Zwan, Martin Haring, Emelie Braun, Lars Borm, Gioele La Manno, Simone Codeluppi, Alessandro Furlan, Nathan Skene, Kenneth D Harris, Jens Hjerling Leffler, Ernest Arenas, Patrik Ernfors, Ulrika Marklund, Sten Linnarsson</span>.
  <a href="https://NathanSkene.github.io/public/publication/moleculararch/" itemprop="name">Molecular architecture of the mouse nervous system</a>.
  Cell,
  2018.
  <p>




<a class="btn btn-primary btn-outline btn-xs" href="https://www.sciencedirect.com/science/article/pii/S009286741830789X?via%3Dihub" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    F Zhu, M Cizeron, Z Qiu, R Benavides-Piccione, M Kopanitsa, N Skene, J DeFelipe, E Fransén, N Komiyama, S Grant</span>.
  <a href="https://NathanSkene.github.io/public/publication/architecture-of-the-mouse-brain-synaptome/" itemprop="name">Architecture of the Mouse Brain Synaptome</a>.
  Neuron,
  2018.
  <p>




<a class="btn btn-primary btn-outline btn-xs" href="https://www.cell.com/neuron/fulltext/S0896-6273%2818%2930581-6" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    Jeanne E. Savage, Philip R. Jansen, Sven Stringer, Kyoko Watanabe, Julien Bryois, Christiaan A. de Leeuw, Mats Nagel, Swapnil Awasthi, Peter B. Barr, Jonathan R. I. Coleman, Katrina L. Grasby, Anke R. Hammerschlag, Jakob A. Kaminski, Robert Karlsson, Eva Krapohl, Max Lam, Marianne Nygaard, Chandra A. Reynolds, Joey W. Trampush, Hannah Young, Delilah Zabaneh, Sara Hägg, Narelle K. Hansell, Ida K. Karlsson, Sten Linnarsson, Grant W. Montgomery, Ana B. Muñoz-Manchado, Erin B. Quinlan, Gunter Schumann, Nathan G. Skene, Bradley T. Webb, Tonya White, Dan E. Arking, Dimitrios Avramopoulos, Robert M. Bilder, Panos Bitsios, Katherine E. Burdick, Tyrone D. Cannon, Ornit Chiba-Falek, Andrea Christoforou, Elizabeth T. Cirulli, Eliza Congdon, Aiden Corvin, Gail Davies, Ian J. Deary, Pamela DeRosse, Dwight Dickinson, Srdjan Djurovic, Gary Donohoe, Emily Drabant Conley, Johan G. Eriksson, Thomas Espeseth, Nelson A. Freimer, Stella Giakoumaki, Ina Giegling, Michael Gill, David C. Glahn, Ahmad R. Hariri, Alex Hatzimanolis, Matthew C. Keller, Emma Knowles, Deborah Koltai, Bettina Konte, Jari Lahti, Stephanie Le Hellard, Todd Lencz, David C. Liewald, Edythe London, Astri J. Lundervold, Anil K. Malhotra, Ingrid Melle, Derek Morris, Anna C. Need, William Ollier, Aarno Palotie, Antony Payton, Neil Pendleton, Russell A. Poldrack, Katri Räikkönen, Ivar Reinvang, Panos Roussos, Dan Rujescu, Fred W. Sabb, Matthew A. Scult, Olav B. Smeland, Nikolaos Smyrnis, John M. Starr, Vidar M. Steen, Nikos C. Stefanis, Richard E. Straub, Kjetil Sundet, Henning Tiemeier, Aristotle N. Voineskos, Daniel R. Weinberger, Elisabeth Widen, Jin Yu, Goncalo Abecasis, Ole A. Andreassen, Gerome Breen, Lene Christiansen, Birgit Debrabant, Danielle M. Dick, Andreas Heinz, Jens Hjerling-Leffler, M. Arfan Ikram, Kenneth S. Kendler, Nicholas G. Martin, Sarah E. Medland, Nancy L. Pedersen, Robert Plomin, Tinca J. C. Polderman, Stephan Ripke, Sophie van der Sluis, Patrick F. Sullivan, Scott I. Vrieze, Margaret J. Wright &amp; Danielle Posthuma</span>.
  <a href="https://NathanSkene.github.io/public/publication/savage-iq-2018/" itemprop="name">Genome-wide association meta-analysis in 269,867 individuals identifies new genetic and functional links to intelligence</a>.
  Nature Genetics,
  2018.
  <p>




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41588-018-0151-7" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    Mats Nagel, Philip R. Jansen, Sven Stringer, Kyoko Watanabe, Christiaan A. de Leeuw, Julien Bryois, Jeanne E. Savage, Anke R. Hammerschlag, Nathan G. Skene, Ana B. Muñoz-Manchado, 23andMe Research Team, Tonya White, Henning Tiemeier, Sten Linnarsson, Jens Hjerling-Leffler, Tinca J. C. Polderman, Patrick F. Sullivan, Sophie van der Sluis &amp; Danielle Posthuma</span>.
  <a href="https://NathanSkene.github.io/public/publication/nagal-neuroticism-2018/" itemprop="name">Meta-analysis of genome-wide association studies for neuroticism in 449,484 individuals identifies novel genetic loci and pathways</a>.
  Nature Genetics,
  2018.
  <p>




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41588-018-0151-7" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    Kenneth D Harris, Hannah Hochgerner, Nathan G Skene, Lorenza Magno, Linda Katona, Carolina Bengtsson Gonzales, Peter Somogyi, Nicoletta Kessaris, Sten Linnarsson, Jens Hjerling-Leffler</span>.
  <a href="https://NathanSkene.github.io/public/publication/kenharris-2018-ca1/" itemprop="name">Classes and continua of hippocampal CA1 inhibitory neurons revealed by single-cell transcriptomics</a>.
  PLoS biology,
  2018.
  <p>




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41588-018-0151-7" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    Nathan G. Skene, Julien Bryois, Trygve E. Bakken, Gerome Breen, James J Crowley, Héléna A Gaspar, Paola Giusti-Rodriguez, Rebecca D Hodge, Jeremy A. Miller, Ana Muñoz-Manchado, Michael C O’Donovan, Michael J Owen, Antonio F Pardiñas, Jesper Ryge, James T R Walters, Sten Linnarsson, Ed S. Lein, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, Patrick F Sullivan, Jens Hjerling-Leffler</span>.
  <a href="https://NathanSkene.github.io/public/publication/skene-natgen-2018/" itemprop="name">Genetic identification of brain cell types underlying schizophrenia</a>.
  Nature Genetics,
  2018.
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2017/06/02/145466" target="_blank" rel="noopener">
  Preprint
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41588-018-0129-5.pdf" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    Jonathan R. I. Coleman, Julien Bryois, Héléna A. Gaspar, Philip R. Jansen, Jeanne E. Savage, Nathan Skene, Robert Plomin, Ana B. Muñoz-Manchado, Sten Linnarsson, Greg Crawford, Jens Hjerling-Leffler, Patrick F. Sullivan, Danielle Posthuma, Gerome Breen</span>.
  <a href="https://NathanSkene.github.io/public/publication/geromeiq-molpsy-2018/" itemprop="name">Biological annotation of genetic loci associated with intelligence in a meta-analysis of 87,740 individuals</a>.
  Molecular Psychiatry,
  2018.
  <p>




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41380-018-0040-6" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="author">
    I Jansen, J Savage, K Watanabe, J Bryois, D Williams, S Steinberg, J Sealock, I Karlsson, S Hagg, L Athanasiu, N Voyle, P Proitsi, A Witoelar, S Stringer, D Aarsland, I Almdahl, F Andersen, S Bergh, F Bettella, S Bjornsson, A Braekhus, G Brathen, C Leeuw, R Desikan, S Djurovic, L Dumitrescu, T Fladby, T Homan, P Jonsson, A Rongve, I Saltvedt, S Sando, G Selbak, N Skene, J Snaedal, E Stordal, I Ulstein, Y Wang, L White, J Hjerling-Leffler, P Sullivan, W van der Flier, R Dobson, L Davis, H Stefansson, K Stefansson, N Pedersen, S Ripke, O Andreassen, D Posthuma</span>.
  <a href="https://NathanSkene.github.io/public/publication/gwasalzh/" itemprop="name">Genetic meta-analysis identifies 9 novel loci and functional pathways for Alzheimers disease risk</a>.
  bioRxiv,
  2018.
  <p>




<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/02/22/258533" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="https://NathanSkene.github.io/public/project/example-external-project/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://example.org" target="_blank" rel="noopener">
  Custom Link
</a>

</p>
</div>

      
    

  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="contact" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Contact</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    <ul class="fa-ul" itemscope>

      
      <li>
        <i class="fa-li fa fa-envelope fa-2x" aria-hidden="true"></i>
        <span id="person-email" itemprop="email"><a href="mailto:nathan.skene@ki.se">nathan.skene@ki.se</a></span>
      </li>
      

      

      

      

      

      

      

      

    </ul>

    

  </div>
</div>

    </div>
  </section>
  



<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Powered by the
      <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
      <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    
    <script src="/public/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({ tex2jax: { inlineMath: [['$','$'], ['\\(','\\)']] } });
    </script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_CHTML" integrity="sha512-tOav5w1OjvsSJzePRtt2uQPFwBoHt1VZcUq8l8nm5284LEKE9FSJBQryzMBzHxY5P0zRdNqEcpLIRVYFNgu1jw==" crossorigin="anonymous"></script>
    
    

  </body>
</html>


